src/actions/layers/overlay.ts

```diff
@@ -0,0 +1,28 @@
+import {IOverlayAction} from "./IOverlayAction";
+import Action from "../Action";
+import {ImageLayer} from "./Sources";
+
+
+
+class Overlay extends Action implements IOverlayAction {
+  imageLayer: ImageLayer;
+  constructor(tImage: ImageLayer) {
+    super();
+    this.imageLayer = tImage;
+  }
+
+  toString(){
+    return `l_${this.imageLayer.asset.publicID}/${this.imageLayer.toString()}/fl_layer_apply`;
+  }
+}
+
+/**
+ * @return {Overlay}
+ */
+function addImageLayer(imageSource: ImageLayer, a?:any, b?:any): Overlay {
+  return new Overlay(imageSource);
+}
+
```

Comment:
Worth refactoring to something like this
`.overlay(overlayLayer(
			  Source.image('cloudinary_icon')
          .adjust(Adjust.opacity(50))
          .resize(Resize.scale(60))
        )
      );`